sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonScanner.java

```diff
@@ -343,19 +340,19 @@ private void saveMetricOnFile(InputFile inputFile, Metric<Integer> metric, Integ
   }
 
   private void handleQuickFixes(InputFile inputFile, RuleKey ruleKey, NewIssue newIssue, PreciseIssue preciseIssue) {
-    if (isInSonarLint(context) && newIssue instanceof NewSonarLintIssue) {
+    if (isInSonarLint(context)) {
       List<PythonQuickFix> quickFixes = preciseIssue.quickFixes();
-      addQuickFixes(inputFile, ruleKey, quickFixes, (NewSonarLintIssue) newIssue);
+      addQuickFixes(inputFile, ruleKey, quickFixes, newIssue);
     }
   }
 
-  private static void addQuickFixes(InputFile inputFile, RuleKey ruleKey, Iterable<PythonQuickFix> quickFixes, NewSonarLintIssue sonarLintIssue) {
+  private static void addQuickFixes(InputFile inputFile, RuleKey ruleKey, Iterable<PythonQuickFix> quickFixes, NewIssue sonarLintIssue) {
     try {
       for (PythonQuickFix quickFix : quickFixes) {
-        NewQuickFix newQuickFix = sonarLintIssue.newQuickFix()
+        var newQuickFix = sonarLintIssue.newQuickFix()
           .message(quickFix.getDescription());
 
-        NewInputFileEdit edit = newQuickFix.newInputFileEdit().on(inputFile);
+        var edit = newQuickFix.newInputFileEdit().on(inputFile);
```

Comment:
I am not sure I understand the reason for the type change here. Is it related to the migration?